app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -71,11 +71,6 @@
     @Before
     public void signIn(){
         Navigation.profileField = false;
```

Comment:
Yes it seems like this technique simplifies a lot the process of "enabling the profile field of the user" or not! (Especially since now the same single variable is used everywhere in our activities)